docs/ReleaseNotes.md

```diff
@@ -39,6 +39,9 @@ The Guava dependency version has been updated to 31.1. Projects may need to chec
 // end next release
 -->
 
+### 3.3.366.0
+* **Bug fix** Don't write to disabled Synthetic indexes [(Issue #2099)](https://github.com/FoundationDB/fdb-record-layer/issues/2099)
```

Comment:
I think this still isn't quite right. This needs to go in the commented out section, replacing one of the bullets that is already there. The release process generates the appropriate bullet point with the correct version number (which probably will be 3.3.366.0, but there's no guarantee). You can see an example of how that's done here, for instance: https://github.com/FoundationDB/fdb-record-layer/pull/2064/files

Reply:
oof. got it